packages/graphql/tests/integration/aggregations/top_level/id.int.test.ts

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Driver } from "neo4j-driver";
+import { graphql } from "graphql";
+import { generate } from "randomstring";
+import neo4j from "../../neo4j";
+import { Neo4jGraphQL } from "../../../../src/classes";
+
+describe("aggregations-top_level-id", () => {
+    let driver: Driver;
+
+    beforeAll(async () => {
+        driver = await neo4j();
+    });
+
+    afterAll(async () => {
+        await driver.close();
+    });
+
+    test("should return the shortest of node properties", async () => {
+        const session = driver.session();
+
+        const typeDefs = `
+            type Movie {
+                testId: ID
+                id: ID
+            }
+        `;
+
+        const id = generate({
+            charset: "alphabetic",
+            readable: true,
+        });
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        try {
+            await session.run(
+                `
+                    CREATE (:Movie {testId: $id, id: 1})
+                    CREATE (:Movie {testId: $id, id: 2})
+                    CREATE (:Movie {testId: $id, id: 3})
+                    CREATE (:Movie {testId: $id, id: 4})
```

Comment:
When ID values are inserted through our API, they are inserted as strings. Probably worthwhile doing the same in these tests, as well as giving them variable lengths so that we can actually validate the behaviour.

Reply:
Yes!